### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by Gelma**
This is a first/quick check. There are more typos.

If this kind of patch is good for you, I can finish it.

### CODE REVIEW

**Comment by Gelma** [comment]
No problem to fix build.
Just let me know if you are interested.

**Comment by Gelma** [comment]
> Thank you, I agree it would be nice to have these errors fixed.

Perfect. So I finish to check the sources.

Thanks a lot for your review!

File: module/zfs/dmu_send.c [hunk 1]
```diff
@@ -1034,7 +1034,7 @@ do_dump(dmu_send_cookie_t *dscp, struct send_range *range)
 		if (err != 0) {
 			if (zfs_send_corrupt_data &&
 			    !dscp->dsc_dso->dso_dryrun) {
-				/* Send a block filled with 0x"zfs badd bloc" */
+				/* Send a block filled with 0x"zfs badd block" */
```
[the remark this answers is not in the document]
Comment (Gelma) [reply module/zfs/dmu_send.c]: Good catch. Reverted. Thanks a lot!

File: tests/zfs-tests/tests/functional/cli_root/zfs_copies/zfs_copies_002_pos.ksh [hunk 2]
```diff
@@ -92,7 +92,7 @@ for val in 1 2 3; do
 	check_used $used $val
 done
 
-log_note "Verify df(1M) can corectly display the space charged."
+log_note "Verify df(1M) can correctly display the space charged."
```
[the remark this answers is not in the document]
Comment (Gelma) [reply tests/zfs-tests/tests/functional/cli_root/zfs_copies/zfs_copies_002_pos.ksh]: uhm... I'm not sure. Looking at the contest of the script, I guess it is really checking the space "charged" to the filesystem.
But I could be wrong, so you tell me.

File: tests/zfs-tests/tests/functional/inuse/inuse_008_pos.ksh [hunk 3]
```diff
@@ -69,7 +69,7 @@ function verify_assertion #slices
 		echo "y" | newfs -v $t > /dev/null 2>&1
 		(( $? !=0 )) && \
 			log_fail "newfs over exported pool " \
-				"failes unexpected."
+				"fails unexpected."
```
[the remark this answers is not in the document]
Comment (Gelma) [reply tests/zfs-tests/tests/functional/inuse/inuse_008_pos.ksh]: Done!

I'm going to push fixes.

**Comment by ikozhukhov** [comment]
i think, split a big update to smaller - based per component - will be much more easy for review and updates

**Comment by Gelma** [comment]
> I'm happy to have these spelling errors / typos fixed! I agree that if there are a huge number of them, it might be easier to review one component at a time (e.g. scripts, libraries, kernel, makefiles, ...)

Oky. Well, let's do this way. In this PR let's the chaos play. Just to have a big overview. Once I have feedback, I will start to submit little patches by groups.

Thanks a lot,
Gelma

**Comment by Gelma** [comment]
> Breaking this up into separate PRs per section is a great idea. When you do, be sure your commits have the "Signed-off-by:" line (see https://github.com/zfsonlinux/zfs/blob/master/.github/CONTRIBUTING.md#commit-message-formats) so we can be sure the rest of the style checks pass. Specifically, I'm wondering if any lines became too long.

ok, thanks a lot!

**Comment by Gelma** [comment]
> i think, split a big update to smaller - based per component - will be much more easy for review and updates

Yes, after initial big review, I'm going to re-submit in the right order/group.

File: tests/zfs-tests/tests/functional/cli_root/zfs_get/zfs_get_005_neg.ksh [hunk 4]
```diff
@@ -56,7 +56,7 @@ typeset  userquota_props=(userquota@root groupquota@root userused@root \
 typeset val_props=("${v_props[@]}" "${userquota_props[@]}")
 set -f	# Force shell does not parse '?' and '*' as the wildcard
 typeset inval_opts=(P R h ? *)
-typeset inval_props=(Type 0 ? * -on --on readonl time USED RATIO MOUNTED)
+typeset inval_props=(Type 0 ? * -on --on readonly time USED RATIO MOUNTED)
```
[the remark this answers is not in the document]
Comment (Gelma) [reply tests/zfs-tests/tests/functional/cli_root/zfs_get/zfs_get_005_neg.ksh]: Reverted.

File: module/zfs/spa.c [hunk 5]
```diff
@@ -9278,7 +9278,7 @@ EXPORT_SYMBOL(spa_inject_delref);
 EXPORT_SYMBOL(spa_scan_stat_init);
 EXPORT_SYMBOL(spa_scan_get_stats);
 
-/* device maniion */
+/* device maniion (just to spot it) */
```
[the remark this answers is not in the document]
Comment (Gelma) [reply module/zfs/spa.c]: Oky, I'm going to take this!

File: module/zfs/dmu_objset.c [hunk 6]
```diff
@@ -1028,7 +1028,7 @@ dmu_objset_create_impl_dnstats(spa_t *spa, dsl_dataset_t *ds, blkptr_t *bp,
 
 	/*
 	 * We don't want to have to increase the meta-dnode's nlevels
-	 * later, because then we could do it in quescing context while
+	 * later, because then we could do it in queiscing context while
```
[the remark this answers is not in the document]
Comment (Gelma) [reply module/zfs/dmu_objset.c]: Thanks! Done.

File: module/zfs/arc.c [hunk 7]
```diff
@@ -6844,7 +6844,7 @@ arc_release(arc_buf_t *buf, void *tag)
 			VERIFY(!arc_buf_is_shared(lastbuf));
 
 			/*
-			 * First, sever the block sharing relationship between
+			 * First, serve the block sharing relationship between
```
[the remark this answers is not in the document]
Comment (Gelma) [reply module/zfs/arc.c]: Reverted.

File: module/zfs/vdev_raidz.c [hunk 8]
```diff
@@ -2120,7 +2120,7 @@ vdev_raidz_io_done(zio_t *zio)
 	 *	3. perform combinatorial reconstruction
 	 *
 	 * Each phase is progressively both more expensive and less likely to
-	 * occur. If we encounter more errors than we can repair or all phases
+	 * occur. If we encounter more errors then we can repair or all phases
```
[the remark this answers is not in the document]
Comment (Gelma) [reply module/zfs/vdev_raidz.c]: Reverted!

File: module/zfs/vdev_label.c [hunk 9]
```diff
@@ -706,7 +706,7 @@ vdev_config_generate(spa_t *spa, vdev_t *vd, boolean_t getstats,
 
 /*
  * Generate a view of the top-level vdevs.  If we currently have holes
- * in the namespace, then generate an array which contains a list of holey
+ * in the namespace, then generate an array which contains a list of holes
```
[the remark this answers is not in the document]
Comment (Gelma) [reply module/zfs/vdev_label.c]: Reverted!

File: module/zfs/dsl_dataset.c [hunk 10]
```diff
@@ -65,7 +65,7 @@
 
 /*
  * The SPA supports block sizes up to 16MB.  However, very large blocks
- * can have an impact on i/o latency (e.g. tying up a spinning disk for
+ * can have an impact on i/o latency (e.g. trying up a spinning disk for
```
[the remark this answers is not in the document]
Comment (Gelma) [reply module/zfs/dsl_dataset.c]: Reverted!

File: module/zfs/abd.c [hunk 11]
```diff
@@ -67,7 +67,7 @@
  *
  *  (1) They avoid use of kmem_*, preventing performance problems where running
  *      kmem_reap on very large memory systems never finishes and causes
- *      constant TLB shootdowns.
+ *      constant TLB shutdowns.
```
[the remark this answers is not in the document]
Comment (Gelma) [reply module/zfs/abd.c]: Reverted!

File: module/icp/include/sys/crypto/sched_impl.h [hunk 12]
```diff
@@ -448,13 +448,13 @@ typedef struct kcf_ntfy_elem {
...
  * at least one product) and a kernel client may come from different
- * low-priority interrupt levels. We will have CYRPTO_TASKQ_MIN number
+ * low-priority interrupt levels. We will have CRYPTO_TASKQ_MIN number
  * of cached taskq entries. The CRYPTO_TASKQ_MAX number is based on
  * a throughput of 1GB/s using 512-byte buffers. These are just
  * reasonable estimates and might need to change in future.
  */
 #define	CRYPTO_TASKQ_THREADS	8
-#define	CYRPTO_TASKQ_MIN	64
+#define	CRYPTO_TASKQ_MIN	64
```
[the remark this answers is not in the document]
Comment (Gelma) [reply module/icp/include/sys/crypto/sched_impl.h]: Yep. Done in another commit.

File: module/zfs/vdev.c [hunk 13]
```diff
@@ -4191,7 +4191,7 @@ vdev_space_update(vdev_t *vd, int64_t alloc_delta, int64_t defer_delta,
 	 * Apply the inverse of the psize-to-asize (ie. RAID-Z) space-expansion
 	 * factor.  We must calculate this here and not at the root vdev
 	 * because the root vdev's psize-to-asize is simply the max of its
-	 * childrens', thus not accurate enough for us.
+	 * children', thus not accurate enough for us.
```
[the remark this answers is not in the document]
Comment (Gelma) [reply module/zfs/vdev.c]: Done!

**Comment by Gelma** [comment]
Now I submit the first small set (as separate PR). If is good for you guys, I'm going to submit the rest.

**Comment by Gelma** [comment]
I'm going to close this PR. I've finished to submit the chopped version.

Thanks you all,
Gelma

**Pull request closed without merging**